react-components/src/components/Reveal3DResources/useCalculateCadStyling.tsx

```diff
@@ -13,7 +13,7 @@ import {
   useFdmAssetMappings,
   useMappedEdgesForRevisions
 } from '../CacheProvider/NodeCacheProvider';
-import { useMemo } from 'react';
+import { useEffect, useMemo } from 'react';
```

Comment:
are we using useEffect in this file? I cant find it

Reply:
Removed :see_no_evil: